packages/features/ee/round-robin/roundRobinReassignment.ts

```diff
@@ -151,17 +151,6 @@ export const roundRobinReassignment = async ({
     reassignedHost: reassignedRRHost,
   });
 
-  // Assume the RR host was labelled as a team member
-  if (reassignedRRHost.email !== organizer.email) {
-    teamMembers.push({
-      id: reassignedRRHost.id,
-      email: reassignedRRHost.email,
-      name: reassignedRRHost.name || "",
-      timeZone: reassignedRRHost.timeZone,
-      language: { translate: reassignedRRHostT, locale: reassignedRRHost.locale ?? "en" },
-    });
-  }
```

Comment:
The new host was shown double in the email. @sean-brydon can you make sure this doesn't break anything?